src/PgManyToManyRelationPlugin.ts

```diff
@@ -67,7 +67,7 @@ export const PgManyToManyRelationPlugin: GraphileConfig.Plugin = {
             junctionTable.getRelation(rightRelationName).extensions,
             rightTable.extensions,
           ]);
-          return ["manyToMany", "connection", "list", behavior, overrides];
+          return ["manyToMany", behavior, overrides];
```

Comment:
@benjie , there's probably a way to keep these default behaviors with lower precedence than the behaviors added by the [V4 plugin ](https://github.com/graphile/crystal/blob/b9c4e898635181e31e2e0f06d6f1f327450f2c3d/postgraphile/postgraphile/src/presets/v4.ts#L124), but it wasn't clear to me how best to do that.